types/entity.go

```diff
@@ -33,10 +34,12 @@ func (e Entity) MarshalJSON() ([]byte, error) {
 		UID        ImplicitlyMarshaledEntityUID   `json:"uid"`
 		Parents    []ImplicitlyMarshaledEntityUID `json:"parents"`
 		Attributes Record                         `json:"attrs"`
+		Tags       Record                         `json:"tags"`
```

Comment:
@patjakdev did you mention that for Rust parity this should be `omitempty` ?

Reply:
I guess, either way, a test showing it does what we want it to do for the empty tags case would be nice.

Reply:
(I see a test exists for this, so that's nice.)

Reply:
Yes, we deliberated on this here: https://github.com/strongdm/cedar-go/pull/11#discussion_r1870037794